src/configuration.rkt

```diff
@@ -80,3 +80,5 @@
 ;; where is the runtime to be used located
 (define runtime-path : (Parameterof (Option Path)) (make-parameter #f))
 (define hashcons-path : (Parameterof (Option Path)) (make-parameter #f))
+(define constants-path : (Parameterof Path)
+  (make-parameter (build-path "src/backend-c/runtime/constants.h")))
```

Comment:
This path should be defined with define-runtime-path so that any uses of it are position independent. Also does it make sense to have this as a parameter? I can't think of a time when we would want to change the constants for any particular runtime.

Reply:
Done